app/helpers/batch_helper.rb

```diff
@@ -98,52 +128,68 @@ def batch_raw_data_table(batch)
   # Temporary method helpful for debugging during development
```

Comment:
We should remove this comment from any temp method that now has a permanent home in the admin console